red/df1.js

```diff
@@ -202,6 +212,11 @@ module.exports = function (RED) {
 
         function removeListeners() {
              if (df1 !== null) {
+                that.removeListener('connected',onConnect)      
+                that.removeListener('disconnected',onDisconnect) 
+                that.removeListener('error',onError) 
+                that.removeListener('timeout',onTimeout)           
+

```

Comment:
I believe those lines here are not needed. Is there any specific reason for those `removeListener`s to be added here?
As far as I can see there are no callbacks set for those events on the `this/that` variable (e.g. there are no `that.on('error')` to be found on this code)

Reply:
really weren't necessary, I removed it in the last commit.